tensorflow_io/bigquery/processing_census_example.ipynb

```diff
@@ -0,0 +1,557 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "processing census using BigQuery TensorFlow connector.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "view-in-github"
+      },
+      "source": [
+        "\u003ca href=\"https://colab.research.google.com/github/tensorflow/io/blob/master/tensorflow_io/bigquery/processing_census_example.ipynb\" target=\"_parent\"\u003e\u003cimg src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/\u003e\u003c/a\u003e\n"
```

Comment:
For other tensorflow sites we have a standard button bar. Could you delete this colab button and copy the bar from this file (and adjust the links)?

https://github.com/tensorflow/docs/blob/master/site/en/tutorials/images/classification.ipynb

Reply:
sure, didn't know that there is a process for this, happy to do it right way.